### PR TITLE
VIH-7352 amended design of participants consultation header

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7352

### Change description ###

changes for participant consultation header

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html [hunk 1]
```diff
@@ -115,7 +115,6 @@
           (click)="lockPrivateConsultation(!roomLocked)"
           type="button">
           {{ roomLocked ? 'Unlock the room ' : 'Lock the room ' }}
-          <span class="fas" [class]="roomLocked ? 'fa-lock-alt' : 'fa-lock-open-alt'"></span>
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html:118]: oh, what happened to the little icon :(

**Review by levibutler — COMMENTED** [review]
My diff isnt working correctly worrying, do you have a screenshot of the changes?

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html:118]: New design, sorry :( , they removed it

**Review by levibutler — CHANGES_REQUESTED** [review]
Spoken with Junior and Lorenzo, going to simplify this change a little. Ill stick in a commit later today

**Comment by levibutler** [comment]
New pr for changes

**Pull request closed without merging**